invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/BookDetails/components/BookMetadata/components/RequestNewLoanForm/state/actions.js

```diff
@@ -7,11 +7,12 @@ import {
 } from '../../../../../../../../common/components/Notifications';
 
 export const requestNewLoanForBook = (docPid, loan) => {
```

Comment:
Maybe we should consider object literal as parameters, to provide an example when I see `requestNewLoanForBook('123', {..})` I have no clue what `123` is, I could guess but I am not very good at guessing. We would have something like:

```
requestNewLoanForBook({
    document_pid: '123',
    loan: {...}
})
```

Reply:
I've created a ticket for that to discuss